.gitignore

```diff
@@ -31,3 +31,4 @@
 
 ### driftctl
 /.driftignore
+.driftctl
```

Comment:
This can be removed

Reply:
I added it because terraform reader tests were creating a `.driftctl` folder within codebase and I committed it by mistake multiple times

Reply:
We should investigate why test are creating theses folders instead of just ignoring the consequences. Can you create a maintenance issue for that ?